libs/mgjson/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mgjson
+PKG_RELEASE:=1
+
+PKG-BRANCH=main
```

Comment:
This variable is for what? How it is used?

Reply:
It was in the sample i started with. I wasn't sure of its significance, and again, could not get an answer as to whether or not it was required